fabric-biomes-v1/src/testmod/java/net/fabricmc/fabric/test/biome/FabricBiomeTest.java

```diff
@@ -31,8 +33,15 @@
 		TestCrimsonForestBiome biome = Registry.register(Registry.BIOME, new Identifier(MOD_ID, "test_crimson_forest"), new TestCrimsonForestBiome());
 		NetherBiomes.addNetherBiome(Biomes.BEACH);
 		NetherBiomes.addNetherBiome(biome);
+
+		TestEndHighlandsBiome testEndBiome = Registry.register(Registry.BIOME, new Identifier(MOD_ID, "test_end_highlands"), new TestEndHighlandsBiome());
+		EndBiomes.addHighlandsBiome(testEndBiome, 5.0);
 	}
 
 	public class TestCrimsonForestBiome extends CrimsonForestBiome {
 	}
+
+	public class TestEndHighlandsBiome extends EndHighlandsBiome {
+
+	}
```

Comment:
```suggestion
	public class TestEndHighlandsBiome extends EndHighlandsBiome {
	}
```

So checkstyle builds

Reply:
That doesn't prevent it from building.

Reply:
must have been mistaken. I'd say then to be consistent with the other test call